src/main/java/seedu/exercise/logic/commands/SuggestBasicCommand.java

```diff
@@ -15,12 +19,13 @@
     public static final String MESSAGE_SUCCESS = "Listed all suggested basic exercises.";
 
     public static final String MESSAGE_USAGE_SUGGEST_BASIC = "Parameters: "
-            + PREFIX_SUGGEST + "SUGGEST TYPE ";
+            + PREFIX_SUGGEST + "BASIC";
```

Comment:
Seems more apt to leave it as SUGGEST_TYPE. "basic" is like the argument to pass after the prefix alr.

Reply:
I thought the words in CAPS describes what kind of arguments should go in there. E.g. custom s/PREFIX_NAME f/FULL_NAME p/PARAMETER_TYPE. Right now, it feels like "basic" is a type of "BASIC" and "possible" is a kind of "POSSIBLE".

Reply:
But i get where you are coming from. "s/basic" should not expect any more arguments after that but "s/possible" expects "m/" or custom property prefixes. So if can't just provide a prompt like "suggest s/SUGGEST_TYPE" because this wouldn't make sense if suggest type is "possible"

Reply:
Maybe can leave it as such for now. Or can just change to SUGGEST_TYPE, but be more explicit about the additional prefixes needed for each type

Reply:
I get what you mean SUGGEST_TYPE is like a variable where it can take "basic" or "possible". 
Ok, i will make it consistent it with the add command message usage.